content/docs/SUSHI/running/_index.md

```diff
@@ -155,4 +156,46 @@ When running SUSHI, the IG Publisher will look for an optional **fsh.ini** contr
 sushi-version = 0.16.0
 timeout = 120
 ```
-{{% /alert %}}
\ No newline at end of file
+{{% /alert %}}
+
+
+### Preprocessed FSH
+When running SUSHI, the `-p` or `--preprocessed` flag can be used to to create a **_preprocessed** folder in SUSHI's output folder (**customized-ig/_preprocessed** in the example above). This folder will contain representations of the input FSH after several preprocessing steps have taken place. These steps include resolution of [`Alias` values](http://build.fhir.org/ig/HL7/fhir-shorthand/branches/master/reference.html#defining-aliases), insertion of [`RuleSet` rules](http://build.fhir.org/ig/HL7/fhir-shorthand/branches/master/reference.html#defining-rule-sets), and resolution of [soft indexing](http://build.fhir.org/ig/HL7/fhir-shorthand/branches/master/reference.html#soft-indexing). This is mainly provided as a debugging tool, for the author to verify that SUSHI is preprocessing the input FSH in an expected way, and to help trace errors in the output of SUSHI back to their source.
```

Comment:
Consider adding:
> For example, if the IG Publisher reports an error on element `Bundle.entry[56].resource`, it may be difficult to identify the problematic entry in your FSH source if you used soft-indexing. It is much easier, however, to identify the problematic element in the preprocessed FSH that contains explicit indices.

Reply:
Considered, and added in https://github.com/FSHSchool/site/pull/42/commits/2ee12fa49e1511aae4752703f90b96ad259f984f